IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MemberTypePost/DocPropertySet.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MemberTypePost" UniqueId="62bc6424-3d87-4aa3-a031-ae14e1e0bec8" ApplicableType="IfcMember/POST">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NominalHeight_1Fy4XYaJXDTBDDb9f06ofi" />
+		<DocProperty xsi:nil="true" href="ConicityRatio_1q2DJWtf58PvkMOQBrIqOF" />
+		<DocProperty xsi:nil="true" href="LoadBearingCapacity_2RBEe0qSKHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="PrevailingWindVelocity_0jIkU0qV8HuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="PrevailingWindDirection_0dxPq0qV8HuO00025QrEV" />
```

Comment:
dont like to place weather properties directly on products as this data should be on the site usually or attached to spatial objects to denote a "coverage for the data". I would place this data as a work item to be explored properly P&W came up with some initial ideas but not a finished proposal.